source/Introduction/About-SpaceROS.rst

```diff
@@ -1,12 +1,96 @@
+
+.. image:: /images/ros_logo.png
+    :width: 125
+
+Meanwhile, back here on Earth, the broader robotics community has standardized on ROS, an open source platform for building robotics applications.
+ROS includes communications middleware, drivers that interface to hardware, state-of-the-art algorithms, and a wide range of developer tools.
+Implementing a robot for space applications can be greatly simplified by incorporating a robotics framework like ROS.
+For example, ROS packages like `MoveIt2 <https://github.com/ros-planning/moveit2>`_ can plan and execute motion for manipulators, like the Canadarm or the Robonaut with its hands and fingers.
+Other packages like `Navigation2 <https://github.com/ros-planning/navigation2>`_ provide the means for robots to perform dynamic path planning and navigation from point A to point B, while avoiding obstacles.
+
+.. image:: /images/ros-robots.png
+
+.. rst-class:: image-subtitle
+
+    ROS robots are everywhere on Earth
+
+In addition, ROS is much more than just an open source code base.
+There is a thriving ecosystem of developers around the world that use and contribute to ROS.
+ROS is commonly used at the university level for robotics research.
+While at university, students work on ROS-based robots and become familiar with the APIs, tooling, and packages provided by the ROS ecosystem.
+
+.. image:: /images/ros2.png
+    :width: 70
+
+The latest version of ROS, ROS 2, offers a standard software platform built from the ground up for use in production in high reliability and safety critical systems.
+Previously, with ROS 1, prototype code would likely have to be rewritten to target a production system.
+With ROS 2, the goal is to be able to leverage early research and prototypes without having to rewrite the code.
+There are many working groups associated with ROS that help advance it in various ways, including some that are relevant to mission critical systems, like the Safety and Real-Time working groups.
+ROS 2 can use industry standard DDS implementations as its core communications middleware.
+DDS is used for high reliability systems such as air-traffic control, autonomous vehicles, medical devices, and power grid management.
+
+.. image:: /images/logo.svg
+    :width: 250
+
+Space ROS will be an open source, space-certifiable robotics framework that is designed to meet flight software standards.
+It will begin as a subset of ROS 2 and will be aligned with aerospace and NASA standards, such as `DO-178C <https://en.wikipedia.org/wiki/DO-178C>`_ and `NPR7150.2 <https://nodis3.gsfc.nasa.gov/displayDir.cfm?t=NPR&c=7150&s=2>`_, so that it can be adopted for high assurance missions.
+Space ROS will also provide space-specific modules and interface with existing flight software tools and frameworks.
+Space ROS will address the demand for software reuse in space robotics while bringing the benefits of ROS to space.
+
+..
```

Comment:
Again this is causing the paragraph below to be interpreted as a comment 
directive. Intentional?

Reply:
Yes, it was intentional to keep the text around as I was working on it. I'm done with editing (for now) and have removed it.